docs/documents/070_tooling.dox

```diff
@@ -84,9 +84,9 @@ $ cmake \
    -DCMAKE_CXX_CPPCHECK="cppcheck;--enable=all" \
    -DCMAKE_CXX_CPPLINT="cpplint.py" \
    -DCMAKE_CXX_INCLUDE_WHAT_YOU_USE="path/to/iwyu;-p;." \
-   -DPYTHON=ON \
-   -DMPI=ON \
-   -DPETSC=ON \
+   -DPRECICE_PythonActions=ON \
+   -DPRECICE_MPICommunicator=ON \
```

Comment:
The MPI Variable is called `PRECICE_MPICommunication`, not `PRECICE_MPICommunicator`!

Reply:
I just took it from [the PR](https://github.com/precice/precice/pull/609), but good point!

Reply:
Fixed